lib/decorateWithMetrics.js

```diff
@@ -6,6 +6,11 @@ import _ from 'underscore';
  */
 let stats = {};
 
+/* For some reason `performance.now()` does not start from `0` but a very large offset
+* like `508,080,000` see: https://github.com/facebook/react-native/issues/30069
+* Capturing an offset allows us to record start/ends times relative to app launch time */
```

Comment:
Interesting, since it is supposed to be [more reliable than `Date`](https://github.com/Expensify/Expensify.cash/issues/3889#issuecomment-877395187), but still has this issue? 🤔 

Not a blocker, but I'm not sure I'd expect it this to be `0`. `performance.now()` returns elapsed since the time origin (or is supposed to at least).

Reply:
Yeah it will never be `0` but the origin is definitely not app launch the benchmark would say something like "Last call finished at: 3940min" which is useless

Reply:
Got it that makes more sense to me now. Thanks for the explanation.